packages/math-input/src/components/echo-manager.tsx

```diff
@@ -63,9 +63,9 @@ class Echo extends React.Component<EchoProps> {
 
 type EchoPropType = {
     animationId: string;
-    animationType: keyof typeof EchoAnimationTypes;
```

Comment:
Another benefit of moving to enums is that we don't have to do the `keyof typeof` gymnastics!